File: COData-WEB-Desarrollo-isai/FrontEnd-VS-Code/COData-Angular/src/app/models/usuario.ts
export interface Usuario {
  usuarioId: number;
  nombre: string;
  email: string;
  contrasenia: string; // <-- CAMBIADO: De 'contraseña' a 'contrasenia'
  fechaRegistro: Date;
  teléfono?: string;
  rol?: string;
  avatar?: string;
}

File: COData-WEB-Desarrollo-isai/BackEnd-VS-C#/COData-Web_BackEnd/COData-Web_BackEnd/Models/Usuario.cs
namespace COData_Web_BackEnd.Models
{
    public class Usuario
    {
        public int UsuarioId { get; set; }
        public string Nombre { get; set; }
        public string Email { get; set; }
        public string Contraseña  { get; set; }
        public DateTime FechaRegistro { get; set; }
        public string? Teléfono { get; set; }
    }
}
